package.json

```diff
@@ -9,6 +9,7 @@
     "custom-elements-manifest": "npm run custom-elements-manifest --workspaces --if-present && node ./scripts/create-api-tables.mjs",
     "debug": "web-test-runner --watch --config web-test-runner-chrome.config.mjs",
     "debug:firefox": "web-test-runner --watch --config web-test-runner-firefox.config.mjs",
+    "debug:no-scoped-registries-polyfill": "npm run debug -- --no-scoped-registries-polyfill",
```

Comment:
Should we add both `with polyfill` and `without polyfill` configs to `test:browser` so that the `npm run test` cover both cases? Or is it to be done in separate PR in the future?

Reply:
I was thinking about that as well, but for now I kept it limited. Feel free to do a proposal in the future